src/main/java/io/ebeaninternal/extraddl/model/ExtraDdlXmlReader.java

```diff
@@ -20,15 +20,16 @@
   /**
    * Return the combined extra DDL that should be run given the platform name.
    */
-  public static String buildExtra(String platformName) {
+  public static String buildExtra(String platformName, boolean drops) {
 
     ExtraDdl read = ExtraDdlXmlReader.read("/extra-ddl.xml");
     if (read == null) {
       return null;
     }
     StringBuilder sb = new StringBuilder(300);
     for (DdlScript script : read.getDdlScript()) {
-      if (matchPlatform(platformName, script.getPlatforms())) {
+      if (script.isDrop() == drops
```

Comment:
Note to self - review line format

Reply:
What's your line format, line length? Maybe you can add an http://editorconfig.org/ and configure at least
indent_style = space
indent_size = 2
max_line_length = ???

Reply:
Thanks - set it to 130, ta.